plugin/trino-iceberg/pom.xml

```diff
@@ -168,6 +168,12 @@
         </dependency>
 
         <!-- used by tests but also needed transitively -->
+        <dependency>
+            <groupId>io.trino</groupId>
+            <artifactId>trino-parser</artifactId>
+            <scope>runtime</scope>
```

Comment:
exclude instead.

Reply:
The enforcer plugin would fail if I'd do the exclusion.

I've rebased on top of master and did an `undo` for the changes from this PR of adding `trino-parser` as runtime dependency. Note that `trino-parser` is on `master` marked at the moment as a `test` dependency for `trino-iceberg` project.